bitnami/rabbitmq/templates/networkpolicy.yaml

```
@@ -59,6 +59,9 @@ spec:
         - port: {{ .Values.containerPorts.amqpTls }}
         - port: {{ .Values.containerPorts.dist }}
         - port: {{ .Values.containerPorts.manager }}
+        {{- range .Values.extraContainerPorts }}
+        - port: {{ . }}
```

I have noticed that the format of the extraContainerPorts is
```yaml
extraContainerPorts:
- name: new_port_name
  containerPort: 1234
```

Therefore using `{{ . }}` would result in invalid syntax after rendering.

```suggestion
        - port: {{ .containerPort }}
```